packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsNoDataState/TransactionsNoDataState.scss

```diff
@@ -1,23 +1,5 @@
 .wallets-transactions-no-data-state {
-    display: flex;
-    padding-block: 3.2rem 1.6rem;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    gap: 2.4rem;
-
-    @include mobile {
-        padding-block: 0;
-    }
-
-    &__title {
-        color: var(--system-light-1-prominent-text, #333);
-        text-align: center;
-
-        /* desktop/paragraph/P1 - bold */
-        font-size: 1.6rem;
-        font-style: normal;
-        font-weight: 700;
-        line-height: 2.4rem; /* 150% */
+    @include desktop {
+        margin-top: 3.2rem; // 48px + 32px = 80px
     }
```

Comment:
Question: Would this same margin-top with the other locked components also work for the transaction page considering that we have the filter component? 

<img width="633" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/781d719c-a803-4bdb-b1b5-dc2467aa6237">

Reply:
You're right. This should be `margin-top: 2.4rem` for both desktop and mobile.